src/engraving/types/types.h

```diff
@@ -622,11 +622,25 @@ enum class DynamicType : char {
     LAST
 };
 
+//! OBSOLETE. Use VoiceAssignment
```

Comment:
Maybe move to some file in `src/engraving/rw/compat`?

Edit: ah, I see that `compatmidirenderinternal.cpp` still uses it. In that case, let's keep this in mind and potentially create a tech debt issue.